src/pages/markets/instruments/_cryptocurrencies-cfds.tsx

```diff
@@ -15,7 +18,7 @@ const CryptocurrenciesCFDs = () => {
             </EU>
             <ROW>
                 <>
```

Comment:
Remove this empty arrow, not needed as <EU> component is holding only one component